shared/applicability/oval/installed_app_is_ocp4.xml

```diff
@@ -138,7 +138,7 @@
 
   <ind:yamlfilecontent_state id="state_ocp4_{{{ minorversion }}}" version="1">
       <ind:value datatype="record">
-          <field name="#" datatype="string" operation="pattern match">4\.{{{ minorversion }}}.*</field>
+          <field name="#" datatype="string" operation="pattern match">\b4\.{{{ minorversion }}}.*</field>
```

Comment:
The proposed regex can still have multiple matches:
For example: `\b4\.16.*` will match  `4.16` and `4.4.16`

How about adding an anchor to the start of string?
```suggestion
          <field name="#" datatype="string" operation="pattern match">^4\.{{{ minorversion }}}.*</field>
```